### DISCUSSION TITLE
Target OS alignment with BCSC App

### DISCUSSION THREAD

**Discussion by cvarjao** [bcgov/bc-wallet-mobile#2182] [General]
Current Min OS:
- Android [API 21](https://github.com/bcgov/bc-wallet-mobile/blob/main/app/android/build.gradle#L6)
- iOS [12](https://github.com/bcgov/bc-wallet-mobile/blob/main/app/ios/Podfile#L5)

Considerations:
- Market/OS share from [Apple](https://developer.apple.com/support/app-store/)
    - iOS Apple Stats: iOS 16 = 81%, iOS 15=13%
    - [iOS StatCounter](https://gs.statcounter.com/os-version-market-share/ios/mobile-tablet/canada)
    - [Android StatCounter](https://gs.statcounter.com/os-version-market-share/android/mobile-tablet/canada)
- App Attestation requirements [iOS 14.0+](https://developer.apple.com/documentation/devicecheck/dcappattestservice?language=objc)
- Active Supported for [iOS](https://endoflife.date/ios) and [Android](https://endoflife.date/android)
- Security Requirements/Recommendations (TEE/Enclave)
- React Native 0.72 requires [iOS 12.4 and Android 5(API 21) ](https://github.com/facebook/react-native/tree/v0.72.4#-requirements)

**Comment by marcos-carretero** [comment]
BCSC current OS support:

- iOS 13 and up (there is still an outdated version of the BCSC app that is available for iOS 10, 11, 12, but will be disabled at the end of September)
- Android 6 and up 
```
minSdkVersion 23
targetSdkVersion 33
```
